hibernate-core/src/test/java/org/hibernate/test/schemaupdate/SchemaExportSqlServerWithSequenceDefaultSchemaCatalog.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.test.schemaupdate;
+
+import java.util.EnumSet;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.boot.spi.MetadataImplementor;
+import org.hibernate.cfg.Environment;
+import org.hibernate.dialect.SQLServer2012Dialect;
+import org.hibernate.service.ServiceRegistry;
+import org.hibernate.tool.hbm2ddl.SchemaExport;
+import org.hibernate.tool.schema.TargetType;
+
+import org.hibernate.testing.RequiresDialect;
+import org.hibernate.testing.TestForIssue;
+import org.hibernate.testing.junit4.CustomRunner;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Andrea Boriero
+ */
+@TestForIssue(jiraKey = "HHH-XXX")
```

Comment:
Could you fix the `jiraKey` - and  I guess the author, or did Andrea help here?

Reply:
@Sanne, there is no jira key. This was cherry-picked from https://github.com/hibernate/hibernate-orm/commit/aaba4767fe2842c442104dff0a377a0f63ecffdd .

@Andrea and I agreed that it was not necessary to create a new jira issue for this.

I suppose I could fill in "HHH-14835" for the jira key though.

Reply:
Oops, sorry, my mistake. That was not from cherry-picking. I'll fix it...

Reply:
Fixed in https://github.com/hibernate/hibernate-orm/pull/4235/commits/823be540cae36c39b2de0e5f3947f2a246984097 .

Reply:
I copied the test from a test that Andrea wrote, with minor changes.